src/index.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
```

Comment:
It might be implied with the language attribute, but should we add a dir='ltr' attribute?

Reply:
According to research, the dir attribute is only needed if you're mixing ltr and rtl on the same webpage (and even then, it's not always needed, but may help reduce the chance of text-bugs). If we only use a ltr language in our app, the ltr attribute is implicit